lte/gateway/c/core/oai/tasks/mme_app/mme_app_ha.cpp

```diff
@@ -18,7 +18,7 @@
 extern "C" {
 #include "lte/gateway/c/core/oai/common/log.h"
 #include "lte/gateway/c/core/oai/tasks/mme_app/mme_app_ha.hpp"
-#include "lte/gateway/c/core/oai/include/ha_messages_types.h"
+#include "lte/gateway/c/core/oai/include/ha_messages_types.hpp"
```

Comment:
Move this out of extern block

Reply:
Moving this newly converted file was fine. Along with this file, I moved `#include "lte/gateway/c/core/oai/tasks/mme_app/mme_app_ha.hpp"` also out of extern, but that was giving compilation error for undefined reference to function `mme_app_handle_ue_offload` which is used in the file `tasks/mme_app/mme_app_main.c`, therefore I have moved the function `mme_app_handle_ue_offload` prototype under extern c in the file `tasks/mme_app/mme_app_ha.hpp`